realm/realm-library/src/main/cpp/io_realm_internal_Table.cpp

```diff
@@ -505,7 +510,12 @@ JNIEXPORT jint JNICALL Java_io_realm_internal_Table_nativeGetColumnType(JNIEnv*
         return 0;
     }
 
-    return static_cast<jint>(TBL(nativeTablePtr)->get_column_type(S(columnIndex))); // noexcept
+    auto column_type = TBL(nativeTablePtr)->get_column_type(S(columnIndex)); // noexcept
```

Comment:
Is `Table.getColumnType()` used anywhere? I thought we should just delete this.

Reply:
We might use it in the Dynamic API. I didn't check though

Reply:
Used in `MutableRealmObjectSchema` at least.